### PR TITLE
Add runbook

### PR DESCRIPTION
**Pull request by bboykov**
# Description

## What

Add runbook for the service.

## Why

[JIRA ticket](https://financialtimes.atlassian.net/browse/UPPSF-1181).

## Scope and particulars of this PR (Please tick all that apply)

- [ ] Tech hygiene (dependency updating & other tech debt)
- [ ] Bug fix
- [ ] Feature
- [X] Documentation
- [ ] Breaking change
- [X] Minor change (e.g. fixing a typo, adding config)

___
This Pull Request follows the rules described in our [Pull Requests Guide](https://github.com/Financial-Times/upp-docs/tree/master/guides/pr-guide)

### CODE REVIEW

File: runbooks/runbook.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+The Public People API is a micro-service which aims to provide Person and
+related data given a Person identifier.
+
+## Code
+
+public-people-api
+
+## Primary URL
+
+<https://github.com/Financial-Times/public-people-api>
```
Comment (karsov) [inline runbooks/runbook.md:12]: This is a public API, its URL is:
```suggestion
<https://api.ft.com/people>
```

File: runbooks/runbook.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+content
+
+## Known About By
+
+- hristo.georgiev
+- robert.marinov
+- tsvetan.dimitrov
+- elina.kaneva
+- georgi.ivanov
```
Comment (karsov) [inline runbooks/runbook.md:36]: I'm not really sure if the content team knows about this service.

File: runbooks/runbook.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+FullyAutomated
+
+## Failback Process Type
+
+PartiallyAutomated
+
+## Failover Details
+
+The service is deployed in the delivery clusters as a deployment.
```
Comment (karsov) [inline runbooks/runbook.md:69]: The failover details are missing.
```suggestion
The service is deployed in both Delivery clusters. The failover guide for the cluster is located here: https://github.com/Financial-Times/upp-docs/tree/master/failover-guides/delivery-cluster
```

File: runbooks/runbook.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+
+To access the service clients need to provide basic auth credentials.
+To rotate credentials you need to login to a particular cluster and update varnish-auth secrets.
+
+## Monitoring
+
+Pod health:
+
+- <https://upp-prod-delivery-eu.upp.ft.com/__health/__pods-health?service-name=public-people-api>
+- <https://upp-prod-delivery-us.upp.ft.com/__health/__pods-health?service-name=public-people-api>
```
Comment (karsov) [inline runbooks/runbook.md:105]: To make it look like the other runbooks:
```suggestion
Service in UPP K8S delivery clusters:

- Delivery-Prod-EU health: <https://upp-prod-delivery-eu.upp.ft.com/__health/__pods-health?service-name=public-people-api>
- Delivery-Prod-US health: <https://upp-prod-delivery-us.upp.ft.com/__health/__pods-health?service-name=public-people-api>
```

**Review by karsov — CHANGES_REQUESTED** [review]
The dependency on `public-concepts-api` is missing.

File: runbooks/runbook.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
+# UPP - Public People Api
+
+The Public People API is a micro-service which aims to provide Person and
+related data given a Person identifier.
```
Comment (karsov) [inline runbooks/runbook.md:4]: There is a warning in the runbook validator about multiple lines in the description:
```suggestion
The Public People API is a micro-service which aims to provide Person and related data given a Person UUID identifier.
```

**Review by karsov — APPROVED** [review]

**Review by ivan-p-nikolov — APPROVED** [review]

**Pull request merged**